system/src/Grav/Common/Page/Page.php

```diff
@@ -106,16 +106,16 @@ public function __construct()
         /** @var Config $config */
         $config = Grav::instance()['config'];
 
-        $this->taxonomy = [];
-        $this->process = $config->get('system.pages.process');
+        $this->taxonomy  = [];
+        $this->process   = $config->get('system.pages.process');
         $this->published = true;
     }
 
     /**
      * Initializes the page instance variables based on a file
      *
      * @param  \SplFileInfo $file The file information for the .md file that the page represents
-     * @param  string       $extension
```

Comment:
why all the extra space? it was fine before. if you do not know exactly what psr2 is then read up on it [http://www.php-fig.org/psr/psr-2/](here) to ease your work you can install some linters for php aswell as. https://github.com/squizlabs/PHP_CodeSniffer and https://phpmd.org/ are useful. If you are using phpstorm you can easily set up code style and lint automatically from Code->Reformat code. Then you do not have to guess what is wanted from you and let the tools point it out.